src/twisted/python/_setup.py

```diff
@@ -96,6 +96,7 @@
         'pyasn1',
         'cryptography >= 0.9.1',
         'appdirs >= 1.4.0',
+        'pynacl >= 1.0.1',
```

Comment:
@the0id I don't think you're grasping my concern here:

Let's say I work at a company that's using Twisted for SSH, but that has a heavy-weight dependency audit process for each new library that gets deployed.  Now, we're still adopting new dependencies and causing more trouble for people working at such shops; however, new _crypto_ dependencies are often given much tighter scrutiny.  By making this change unconditional, I have to upgrade Twisted and go through the pynacl review process at the exact same time - rather than installing new Twisted immediately and kicking off the review process for pynacl, which, once it gets approved, gives me a new capability.